Bugzilla/Elastic.pm

```diff
@@ -16,6 +16,12 @@ with 'Bugzilla::Elastic::Role::HasIndexName';
 
 sub suggest_users {
     my ($self, $text) = @_;
+
+    unless (Bugzilla->params->{elasticsearch}) {
```

Comment:
If the type => b default => 0 above means 'boolean false', then this is fine. (If it means anything other than 'boolean', though)